tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{27,34,35,36}, flake8
+skip_missing_interpreters = true
+
+[testenv]
+deps = pytest
+       python-dotenv
+commands = py.test {posargs}
+passenv = VCAP_SERVICES
+
+[testenv:flake8]
+deps = flake8
```

Comment:
I got the error when running `tox` locally:


```python
flake8 inst-nodeps: /tmp/sdk-costela/.tox/dist/glpi-0.2.1.zip
ERROR: invocation failed (exit code 1), logfile: /tmp/sdk-costela/.tox/flake8/log/flake8-12.log
ERROR: actionid: flake8
msg: installpkg

    File "/tmp/sdk-costela/.tox/flake8/lib/python3.5/site-packages/setuptools/py27compat.py", line 7, in <module>
      import six
  ImportError: No module named 'six'

```

I some OS maybe `python-six` is not installed by default.
Can you add the `six` in the list of test dependencies, please?


```diff
 [testenv]
-deps = pytest
+deps = six
+       pytest
        python-dotenv

 [testenv:flake8]
-deps = flake8
-commands = flake8 .
\ No newline at end of file
+deps = six
+       flake8
 commands = flake8 .
```